.github/autobuild/get_build_vars.py

```diff
@@ -43,8 +43,9 @@ def get_build_version(jamulus_pro_version):
 
 def set_github_variable(varname, varval):
     print("{}='{}'".format(varname, varval))  # console output
-    print("::set-output name={}::{}".format(varname, varval))
-
+    outputfile = os.getenv('GITHUB_OUTPUT')
+    with open(outputfile, "a") as ghout:
+        ghout.write(f"varname=varval\n")
```

Comment:
Try this:
```suggestion
        ghout.write(f"{{{varname}}}={{{varval}}}\n")
```

Reply:
Does this let python interpret varname as variable?

Reply:
Yep, it should.
```
>>> foo = 1
>>> f'{{{foo}}}'
'{1}'
```

Reply:
Hmm. I think we both initially read the docs in a wrong way ;-). There's no need for "{"

Reply:
It seems to be building now. But I'll change it to the f syntax